packages/block/src/block.ts

```diff
@@ -149,12 +149,14 @@ export class Block {
   }
 
   /**
-   *  Returns a Buffer Array of the raw Buffers of this block, in order.
+   * Returns a Buffer Array of the raw Buffers of this block, in order.
    */
   raw(): BlockBuffer {
     return [
       this.header.raw(),
-      this.transactions.map((tx) => <Buffer[]>tx.raw()),
+      this.transactions.map((tx) =>
+        'transactionType' in tx && tx.transactionType > 0 ? tx.serialize() : tx.raw()
+      ) as Buffer[],
```

Comment:
Yes, this makes sense. I think this is the right thing to do to get the whole raw and serialized data setup consistent again and avoid this unlucky splitting of the `tx.raw()` method.

We just should be aware here that this introduces another tx type induced version dependency between the libraries, so block serialization will now be wrong when an old block version is used upon a tx-type-containing tx release (but so be it).

I guess we minimally need to communicate the set of library versions to be used when using typed txs very clearly if we want to stay in the minor release realm (as we've done with the clique changes, these were still a bit better encapsulated by the `Common` `consensusType` setting though). 🤔 

//cc @cgewecke